src/Symfony/Component/Validator/Constraints/ChoiceValidator.php

```diff
@@ -100,6 +100,7 @@ public function validate($value, Constraint $constraint)
         } elseif (!\in_array($value, $choices, true)) {
             $this->context->buildViolation($constraint->message)
                 ->setParameter('{{ value }}', $this->formatValue($value))
+                ->setParameter('{{ choices }}', implode(', ', $choices))
```

Comment:
should also be added to the multiple=true case

choices can be any value,  i think we need to map all choices using `formatValue()` also.

Reply:
yep, you're totally right - i've added to the multiple=true case and formatted the value